app/screens/VenueScreen/SponsorCard.tsx

```diff
@@ -37,23 +37,23 @@ export const SponsorCard = ({
   sponsorImages,
   tier = "Gold",
 }: SponsorCardProps) => {
-  const onPress = () => {
-    Linking.openURL(externalURL)
+  const onPress = (url: string) => {
+    Linking.openURL(url)
```

Comment:
I think we have a util `openLinkInBrowser` for this? but I can't recall if you and I saw some issue with it (canOpenUrl was being weird?) and opt'd not to use it or changed it

Reply:
i think its cleaner to use the util function (and we can always fix the bugs in one spot). updated here: https://github.com/infinitered/ChainReactApp2023/pull/90/commits/5b69b3ba35a3f5288eb4581d3594e04f3b829849